### DISCUSSION TITLE
impossibile scaricare film da CB01

### DISCUSSION THREAD

**Discussion by gattorome** [aclap-dev/video-downloadhelper#692] [Help and Q&A]
### Your question

ho acquistato la licenza di conversione video DownloadHelper per Chrome sperando con questa di riuscire a scaricare film come ho fatto negli ultimi anni....invece tutto inutile..il commento e' questo :

sfortunatamente il download di questo media specifico  non è andato a buon fine. Cerchiamo di supportare più siti internet possibile, e ci aiuterebbe molto se ci invii un report di questo errore!

SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 04e5b980] moov atom not found
[in#0 @ 04e59840] Error opening input: Invalid data found when processing input
Error opening input file https://mixdrop.club/f/gn9xwzd4al09n/2/Atlas_HD_2024_WEBDL_1080p.mp4.
Error opening input files: Invalid data found when processing input

cosa occorre fare per riavere il download helper precedente ?
ho pagato 14 €  sperando di fare quello che si poteva fare prima !
grazie

### Addon & CoApp version and details

Sfortunatamente il download di questo media specifico è andato non è andato a buon fine. Cerchiamo di supportare più siti internet possibile, e ci aiuterebbe molto se ci invii un report di questo errore!

SideDownload error: : [mov,mp4,m4a,3gp,3g2,mj2 @ 04e5b980] moov atom not found
[in#0 @ 04e59840] Error opening input: Invalid data found when processing input
Error opening input file https://mixdrop.club/f/gn9xwzd4al09n/2/Atlas_HD_2024_WEBDL_1080p.mp4.
Error opening input files: Invalid data found when processing input

**Comment by Mattmjs** [comment]
I'm a user like you, the mixdrop video didn't give me the error you got. However here is the problem, I started the download
and it is hardly moving. It is so terribly throttled that it's not worth your time to download from mixdrop.
Just move on and try another site.

**Comment by gattorome** [comment]
Grazie per la risposta……

 

Ciao

 

  Loris

 

Da: Mattmjs ***@***.*** 
Inviato: lunedì 27 maggio 2024 10:48
A: aclap-dev/video-downloadhelper
Cc: gattorome; Author
Oggetto: Re: [aclap-dev/video-downloadhelper] impossibile scaricare film da CB01 (Discussion #692)

 

I'm a user like you, the mixdrop video didn't give me the error you got. However here is the problem, I started the download
and it is hardly moving. It is so terribly throttled that it's not worth your time to download from mixdrop.
Just move on and try another site.
